org.civicrm.bootstrapcivihr/scss/mixins/_spinner.scss

```diff
@@ -0,0 +1,24 @@
+/**
+ * Spinner mixin
+ * $speed Speed to rorate the spinner in seconds
+ * $height Height of the spinner
+ * $height Width of the spinner
+ */
+@mixin spinner($speed: 1s, $height: 32px, $width: 32px) {
+  animation: spin $speed linear infinite;
+  background: url(../img/spinner.svg) no-repeat center center !important;
+  display: block;
+  height: $height !important;
```

Comment:
why `!important`?

Reply:
The core CiviCRM spinner already has defined height and width to be `100%`. So, to override it `!important` is used here.

Reply:
why can't you override it by just using a "heavier" selector? It's not good to have "!important" in a generic mixin just to fix a very specific case

Reply:
> why can't you override it by just using a "heavier" selector?

Since the styles are added inline and there is this only one element and has no nested elements, don't think we can target styles using any selectors leaving behind the only option to use `!important`.
<img width="1324" alt="screen shot 2017-11-20 at 8 37 30 pm" src="https://user-images.githubusercontent.com/6307362/33024378-0626bce0-ce33-11e7-821c-1e9aeb088c69.png">

However, I have refactored the mixin avoiding the use of `!important`. And overriding the size of core civicrm spinner  in specific overriding styles.